endpoints.json

```diff
@@ -5,5 +5,12 @@
     "status": "active",
     "version": "3.0.1",
     "URL": "https://development.ons.io/t/fhir"
+  },
+  {
+    "endpointType": "urn:nuts:endpoint:consent",
+    "identifier": "urn:ietf:rfc:1779:O=Nedap,C=NL,L=Groenlo,CN=nedap_nuts_cordapp_development",
+    "status": "active",
+    "version": "4.1",
```

Comment:
Should `version` be the corda node version, or the cordapp version?

Reply:
we haven't decided yet.... but 0.10.0 seems a more likely candidate, yes.